orttraining/orttraining/python/training/onnxblock/_training_graph_utils.py

```diff
@@ -9,6 +9,29 @@
 from onnxruntime.capi._pybind_state import GradientGraphBuilder, get_optimized_model
 
 
+def _disable_training_mode(model: onnx.ModelProto) -> None:
+    """Disables the training mode of the model by removing the training configuration."""
+
+    def disable_training_mode_dropout(node):
+        # Training mode is the third input of Dropout
+        if len(node.input) > 2:
+            node.input[2] = ""
+
+    def disable_training_mode_batchnorm(node):
+        # Training mode is an attribute of BatchNormalization
+        for attr in node.attribute:
+            if attr.name == "training_mode":
+                attr.i = 0
+
+    ops_to_disable_training_mode_func_map = {
+        "Dropout": disable_training_mode_dropout,
+        "BatchNormalization": disable_training_mode_batchnorm,
```

Comment:
should BatchNormInternal be included also? Oh maybe eval graph comes from exported model directly, not tailored based on training graph?